nf_core/lint.py

```diff
@@ -103,21 +104,59 @@ def pf(file_path):
                 self.warned.append((1, "File not found: {}".format(files)))
 
 
+    def check_docker(self):
+        """minimal tests only"""
+        logging.debug('Checking Dockerfile')
+        fn = os.path.join(self.path, "Dockerfile")
+        content = open(fn).read()
```

Comment:
I think Python discourages the the usage of open() without `with` as people tend to forget to close the open file handler. 

`with open(fn, 'r') as fh: content = fh.read()`

Python will close the handler automatically, even if an exception is thrown.

Reply:
Agree. Will change that

Reply:
Quick question, in the [MethylSeq](https://github.com/ewels/NGI-MethylSeq/tree/nf-core) pipeline there is two Dockerfiles
One is `Dockerfile`, the other one `Dockerfile-bwa`, but they both are at the base of the repo.
In Sarek, all the 16 Dockerfiles are in a different directories in a `containers` directory, but they're all called `Dockerfile`.
Will the test handle such different cases?

Reply:
Or should all Dockerfiles be named `Dockerfile` and placed in a specific path?

Reply:
The setup in MethylSeq is broken - Dockerfiles have to be called `Dockerfile` and in their own directory for automated builds to work (I think). I'm intending to merge the two pipeline scripts and docker files for this pipeline.